main/course_description/index.php

```diff
@@ -72,20 +63,32 @@
 	case 'history':		
 		$course_description_controller->listing(true);
 		break;
-	case 'add'	  :   
-		if (api_is_allowed_to_edit(null,true)) {
-			$course_description_controller->add();
-		}
+	case 'add'	  :
+        if (api_is_platform_admin()) {
+            if (api_is_allowed_to_edit(null,true)) {
```

Comment:
No pasa nada porque es código existente re-indentado, pero recuerda que para PSR-2 (sección 4.4): "there MUST be one space after each comma"